priv/templates/boot_win.eex

```diff
@@ -19,7 +19,7 @@
 @if "%RELEASE_MUTABLE_DIR%"=="" (
   set mutable_dir=%release_root_dir%\var
 ) else (
-  set mutable_dir=%RELEASE_MUTABLE_DIR
+  set mutable_dir=%RELEASE_MUTABLE_DIR%\var
```

Comment:
I'm not sure we need to add `\var` here - the idea is that `RELEASE_MUTABLE_DIR` is already set to something like `path\to\var`, so we don't want to manipulate it. We do so in the "is empty" branch because `<root>\var` is the default when not set.